views/timeline.ejs

```diff
@@ -37,11 +37,16 @@
         <!-- grid section that shows timeline of crits -->
         <div class="timeline">
             <!-- Send a Crit -->
+            <% if (userId === undefined) { %>
+                <div>
+
```

Comment:
More blank divs. If you're just trying to display nothing, just change your logic around and only display the form if the user is logged in.

Like:

```
<% if ( userId ) { %>
<form> ... etc. 

<% } %>
```

No need for an else block if there is nothing to display.